pkg/apis/core/validation/validation.go

```diff
@@ -1114,9 +1114,10 @@ func validateMountPropagation(mountPropagation *core.MountPropagationMode, conta
 	if mountPropagation == nil {
 		return allErrs
 	}
+
+	// MountPropagation is GA and can't be disabled via feature gate
 	if !utilfeature.DefaultFeatureGate.Enabled(features.MountPropagation) {
-		allErrs = append(allErrs, field.Forbidden(fldPath, "mount propagation is disabled by feature-gate"))
-		return allErrs
+		glog.Warning("MountPropagation can't be disabled via feature gate")
```

Comment:
warning here, per request validation, isn't the right place... we'd want to warn at server start if someone set this to false

Reply:
Moved the warning to `kubelet.go`.